pkg/schema/validate_test.go

```diff
@@ -73,7 +73,7 @@ func TestNormalizeFieldName(t *testing.T) {
 	}{
 		{"Lowers", "LOWER", "lower", nil},
 		{"Too short", "", "", ErrFieldNameTooShort},
-		{"Too long", strings.Repeat("a", 31), "", ErrFieldNameTooLong},
+		{"Too long", strings.Repeat("a", 100), "", ErrFieldNameTooLong},
```

Comment:
Did you mean to bring in this change?

Reply:
Yep! This test was broken, but the tests weren't working in CI previously.